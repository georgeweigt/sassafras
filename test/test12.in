* No default data set ;
proc reg ;
model y = x ;

