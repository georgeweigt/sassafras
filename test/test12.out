* No default data set
proc reg;
No data set
